src/utils/helpers.js

```diff
@@ -0,0 +1,38 @@
+// separates an array of objects into the number of desired arrays of objects
+// (data structure provided by graphData) each containing a date and value;
+// converts date strings into date objects - as required for multi-line MG graphs
+export const transformGraphData = (graphData, data) => {
+  const finalData = [];
+  for (const key of Object.keys(graphData)) {
+    for (const item of data) {
+      graphData[key].push({ date: new Date(item.date), value: item[key] });
+    }
+    finalData.push(graphData[key]);
+  }
+  return finalData;
+};
+
+export const sortObjects = (data, sortBy, desc) => {
+  data.sort((a, b) => {
+    const item1 = (desc ? b[sortBy] : a[sortBy]);
+    const item2 = (desc ? a[sortBy] : b[sortBy]);
+
+    if (item1 < item2) {
+      return -1;
+    }
+    if (item1 > item2) {
+      return 1;
+    }
+    return 0;
```

Comment:
I don't think this helper method is really since it's only used in one place. Besides that, the logic could be consderably simplified:

* don't bother with the `desc` parameter (just call reverse if you want it in descending order)
* you can just subtract one element from the other instead of doing the comparison (i.e. `return item1 - item2`)

Reply:
Good point - I'll change it to a one liner.